aten/src/ATen/core/Tensor.h

```diff
@@ -37,6 +38,9 @@ namespace at {
 class Tensor;
 using TensorList = ArrayRef<Tensor>;
 
+struct Quantizer;
+using ConstQuantizerPtr = const c10::intrusive_ptr<Quantizer>&;
```

Comment:
it's a pretty deceptive to put const reference inside the type. Maybe put const& in actual usages?

Reply:
I put it here because this type is used in `native_functions.yaml`, to avoid parsing confusions.. I tried to use "const QuantizerPtr&" before but looks like it does not work very well, that's why I changed it to this.

Reply:
Ok, add a comment to it as const & typedefs are evil :)

Reply:
this will go away when we actually expose `Quantizer` to frontend.